molecules/_dropdown.scss

```diff
@@ -110,7 +110,7 @@ $dropdown-background-color-accent: $earl-gray !default;
     display: block;
     .dropdown-container {
       @include dropdown-container($dropdown-width, $dropdown-background-color, $dropdown-shadow);
-      &.wide {
+      &.mod-wide {
```

Comment:
We aligned on a naming convention before some months.
State classes are prefixed by an `is-` like `is-active` or `is-hovered` and modifier classes which changes the default behavior are prefixed by mod- like `mod-large`